src/graphql/public/schema.graphql

```diff
@@ -29,6 +30,24 @@ type AccountDeletePayload {
   success: Boolean!
 }
 
+input AccountDisableNotificationCategoryForChannelInput {
+  category: NotificationCategory!
+  channel: NotificationChannel!
+}
+
+input AccountDisableNotificationChannelInput {
+  channel: NotificationChannel!
+}
+
+input AccountEnableNotificationCategoryForChannelInput {
+  category: NotificationCategory!
+  channel: NotificationChannel!
```

Comment:
I'm wondering if channel should be mandatory. if it was optional, then the mutation would apply to all channels